modules/apps/batch-engine/batch-engine-test/src/testFunctional/macros/BatchEngine.macro

```diff
@@ -48,6 +48,22 @@ definition {
 		WaitForConsoleTextPresent(value1 = "Processing ${fileName}");
 	}
 
+	macro moveFileToOSGIModulesFolder {
+		Variables.assertDefined(parameterList = "${sourceFilePath},${fileName},${fileNameInConsole}");
+
+		if (!(isSet(destinationPath))) {
+			var liferayHome = PropsUtil.get("liferay.home.dir.name");
+
+			var deployFolder = "${liferayHome}/osgi/modules";
+		}
+
+		AntCommand(
+			locator1 = "build-test.xml",
+			value1 = "move-file -Dfile=${sourceFilePath} -Dtofile=${deployFolder}/${fileName}");
+
+		WaitForConsoleTextPresent(value1 = "STARTED ${fileNameInConsole}");
```

Comment:
I'm thinking that we can update the jar file to make fileNameInConsole be same with fileName